### PR TITLE
promote authz dry-run to alpha

### PR DESCRIPTION
**Pull request by yangminzhu**

### CODE REVIEW

File: features/dry_run_authorization_policy.md [hunk 1]
```diff
@@ -89,35 +89,34 @@ https://istio.io/latest/docs/tasks/security/authorization/authz-dry-run/
...
-- [ ] Initial API review.
+- [x] Initial API review: No API changes for alpha graduation.
 
 **Approvals**
 
-- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
-- [ ] The TOC has reviewed and approved promotion of the feature as part of the
-	roadmap for a release.
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature. (graduation plan approved: https://docs.google.com/document/d/1toiHxec3r0dhoMji2ot2k9RV9Q8W7FcX48Fh6kEdSNc/edit#)
+- [ ] The TOC has reviewed and approved promotion of the feature as part of the roadmap for a release.
```
Comment (yangminzhu) [inline features/dry_run_authorization_policy.md:119]: @myidpt @liminw @lizan Please review, thank you.

**Review by myidpt — APPROVED** [review]

**Comment by yangminzhu** [comment]
@istio/technical-oversight-committee could someone help take a look the PR? thanks.

**Comment by louiscryan** [comment]
Is the user-feedback on the feature captured somewhere ?

**Comment by yangminzhu** [comment]
> Is the user-feedback on the feature captured somewhere ?

@louiscryan 

I think we do not have much written feedback on this feature. We do have some customers asking questions like "how to roll out authorization policies in WARN mode before enforcing" in the slack channel and we pointed them to the dry-run task (https://preliminary.istio.io/latest/docs/tasks/security/authorization/authz-dry-run/) which they later confirmed it is exactly what they are looking for.

**Comment by louiscryan** [comment]
I'm fine with progressing the feature. Get one other TOC member?

**Comment by smawson** [comment]
I'm fine with this progressing as well, do we need a TOC discussion or just approve it here? I do see some items still waiting on PRs to be submitted though so those should go in and be updated here.

**Comment by howardjohn** [comment]
+1 from me as well

**Comment by nrjpoddar** [comment]
LGTM

**Comment by Xunzhuo** [comment]
LGTM from me too.

**Comment by yangminzhu** [comment]
@louiscryan  @smawson @howardjohn @howardjohn  Thanks for the review! 

I have updated the PR for the remaining items, the other API PR has been merged and I have also sent the release note PR (https://github.com/istio/istio/pull/36581), Please feel free to approve or let me know if you have other questions.

**Comment by yangminzhu** [comment]
@louiscryan @smawson @howardjohn @howardjohn could you approve the PR now? Thanks.

**Comment by nrjpoddar** [comment]
LGTM

**Review by howardjohn — APPROVED** [review]

**Pull request merged**